pyccel/stdlib/numpy/numpy_f90.f90

```diff
@@ -0,0 +1,145 @@
+! --------------------------------------------------------------------------------------- !
+! This file is part of Pyccel which is released under MIT License. See the LICENSE file   !
+! or go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. !
+! --------------------------------------------------------------------------------------- !
+
+module numpy_f90
+
+    use ISO_C_BINDING
```

Comment:
As in the generated files it would be better to use `use, intrinsic :: ISO_C_Binding`

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/b4ef1309aa2a48ad51b70675d90333c15ab270d4